docs/src/landing.md

```diff
@@ -0,0 +1,59 @@
+# Welcome to the `ethstaker-deposit-cli`!
+
+The [`ethstaker-deposit-cli`](https://github.com/eth-educators/ethstaker-deposit-cli) is a command-line tool forked from Ethereum's [`staking-deposit-cli`](https://github.com/ethereum/staking-deposit-cli) with more functionality and improved performance.
+
+## Important Concerns
+
+This tool generates and/or utilizes mnemonics and validator keystores. Both are highly sensitive materials and their exposure could lead to the loss or theft of funds. Users must take appropriate measures to secure this information.
```

Comment:
We should add a sentence or two about privacy. Even though someone can keep everything secure by following all the good practices, there is a possibility to leak information that someone might want to keep private such as a validator public key. Even if privacy leakage is likely less severe in the grand scheme of things, it might still be important to some users and we should explain it here. There could be some additional recommendations below to keep everything private (such as not publically display or share the deposit data files).

Reply:
I worked on the wording a bit and added a section. Still not feeling 100% on it but it's improving.